Documentation/coding-guidelines/breaking-change-rules.md

```diff
@@ -194,6 +194,9 @@ successfully bind to that overload, if simply passing an `int` value. However, i
 
 * Adding or removing `static` keyword from a member
 
+* Adding a private field to a struct that previously had no state (unless the type of the added field is also a struct with no state)
+> Definite assignment rules allow use of uninitialized variables so long as the variable type is a stateless struct. If the struct is made stateful, it could cause ‘CS0165: Use of unassigned local variable’ in code that uses it.
```

Comment:
I have recently changed the formatting of this document in https://github.com/dotnet/corefx/pull/31876 to use a style that I think makes more sense where following lines are paragraphs (indented by 4 spaces) inside the bullet point, instead of a blockquote below the bullet point.

Could you update the style of this change to make it consistent with the new rest of the article? Thanks.

The new added code would be:

```md
* Adding a private field to a struct that previously had no state (unless the type of the added field is also a struct with no state)

    Definite assignment rules allow use of uninitialized variables so long as the variable type is a stateless struct. If the struct is made stateful, it could cause ‘CS0165: Use of unassigned local variable’ in code that uses it.
```

Reply:
Certainly, thanks Petr!